docs/sources/alerting/set-up/provision-alerting-resources/edit-provisioned-resources/_index.md

```diff
@@ -0,0 +1,47 @@
+---
+canonical: https://grafana.com/docs/grafana/latest/alerting/set-up/provision-alerting-resources/edit-provisioned-resources/
+description: Export and edit provisioned resources in Grafana
+keywords:
+  - grafana
+  - alerting
+  - alerting resources
+  - provisioning
+  - export
+  - edit
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+menuTitle: Export and edit provisioned resources in Grafana
+title: Export and edit provisioned alerting resources in Grafana
+weight: 400
+---
+
+# Export and edit provisioned alerting resources in Grafana
+
+Export your alerting resources, such as alert rules, contact points, and notification policies as Terraform resources. A new **Modify export** mode for alert rules enables you to edit provisioned alert rules and export a modified version.
```

Comment:
not sure if we should say as Terraform resources as we enable to export also as JSON and YAML format?

Reply:
changed this to reflect all formats

Reply:
I don't see the change

Reply:
woudl help if i pushed:) now you should